R/pvalue_maxcombo.R

```diff
@@ -61,13 +61,13 @@ NULL
 #'
 #' @export
 #'
-pvalue_maxcombo<- function(Z,
+pvalue_maxcombo<- function(z,
                            dummy_var,
                            algorithm = GenzBretz(maxpts = 50000, abseps = 0.00001)){
 
-  ans <- (1 - mvtnorm::pmvnorm(lower = rep(Z$Z %>% min() %>% as.numeric(),
-                                           nrow(Z)),
-                               corr = Z %>%
+  ans <- (1 - mvtnorm::pmvnorm(lower = rep(z$Z %>% min() %>% as.numeric(),
```

Comment:
Could you please also change the `$Z` into `$z`? This `$Z` is an output variable from `sim_fixed_n`. So please also remember to update the `Z` in `sim_fixed_n`.

Reply:
Revised and submit it again. Please check my revisions.